morango/syncsession.py

```diff
@@ -278,7 +302,13 @@ def _close_sync_session(self, sync_session):
 
     def _push_record_chunk(self, serialized_recs):
         # push a chunk of records to the server side
-        return self._request(api_urls.BUFFER, method="POST", data=serialized_recs)
+        data = serialized_recs
+        GZIP = GZIP_BUFFER_POST in self.capabilities
```

Comment:
Suggest `use_gzip` (more semantic wording, and lowercase because it's not a constant)